tools/spectral/ipa/__tests__/eachPathAlternatesBetweenResourceNameAndPathParam.test.js

```diff
@@ -0,0 +1,213 @@
+import testRule from './__helpers__/testRule';
+import { DiagnosticSeverity } from '@stoplight/types';
+
+testRule('xgen-IPA-102-path-alternate-resource-name-path-param', [
+  {
+    name: 'valid paths - api/atlas/v2',
+    document: {
+      paths: {
+        '/api/atlas/v2/resourceName': {
+          post: {},
+          get: {},
```

Comment:
You should be able to remove the methods from these, for testing I think just and empty value for each path is alright (since we're only evaluating the paths)